test/int/dbuser_test.go

```diff
@@ -76,20 +77,6 @@ var _ = Describe("AtlasDatabaseUser", func() {
 			Eventually(testutil.WaitFor(k8sClient, createdProject, status.TrueCondition(status.ReadyType)),
 				20, interval).Should(BeTrue())
 		})
-
```

Comment:
this block was moved from the top of the tests to the specific `Describe` that works with creating multiple users and clusters.